drv.go

```diff
@@ -137,6 +137,174 @@ type connPool struct {
 	serverVersion VersionInfo
 }
 
+func (d *drv) openPool(P *PoolParams) (*connPool, error) {
+
+    // determine key to use for pool
+    poolKey := fmt.Sprintf("%s\t%s\t%s\t%d\t%d\t%d\t%s\t%s\t%s\t%t\t%t\t",
+            P.UserName, P.Password, P.DSN, P.MinSessions, P.MaxSessions,
+            P.SessionIncrement, P.WaitTimeout, P.MaxLifeTime, P.SessionTimeout,
+            P.Heterogeneous, P.EnableEvents)
+    fmt.Printf("Pool key: |%s|\n", poolKey)
```

Comment:
if Log != nil {
    Log("pool key", poolKey)
}

Reply:
I'll make this change, too.